packages/react-instantsearch-hooks-dom/src/ui/__tests__/HierarchicalMenu.test.tsx

```diff
@@ -64,7 +64,7 @@ describe('HierarchicalMenu', () => {
             class="ais-HierarchicalMenu-list"
```

Comment:
According to the code snippet in the [spec](https://instantsearch-css.netlify.app/widgets/hierarchical-menu/), any item that belongs to a parent list should have a class `.ais-HierarchicalMenu-item--parent` even if it doesn't itself contain a child, but this is inconsistent with the description ("the menu list item containing children").

Did you check how this is implemented in InstantSearch.js? I think the usage over there would apply, but in this case let's update to spec to remove any confusion.

Reply:
I did check the implementation in IS.js, the class is only applied if the item has children: https://github.com/algolia/instantsearch.js/blob/master/src/components/RefinementList/RefinementList.tsx#L171-L179.

Reply:
That makes sense. Feel free to update the [specs](https://github.com/algolia/instantsearch-specs) then, so we can avoid second-guessing them.